packages/@glimmer/compiler/lib/template-compiler.ts

```diff
@@ -178,18 +178,13 @@ export default class TemplateCompiler implements Processor<InputOps> {
         this.opcode(['attrSplat'], action);
       } else if (isStatic && !isComponent) {
```

Comment:
@rwjblue I think we should just remove the isComponent here, static attributes don't have XSS even on a component